mobile/test/kotlin/integration/ReceiveTrailersTest.kt

```diff
@@ -84,59 +61,32 @@ class ReceiveTrailersTest {
       RequestHeadersBuilder(
         method = RequestMethod.GET,
         scheme = "https",
-        authority = "example.com",
-        path = "/test"
+        authority = "localhost:${httpTestServer.port}",
+        path = "/simple.txt"
```

Comment:
does the path need to be simple.txt or can it be anything?

Reply:
Yeah, it doesn't matter since the test server always returns HTTP 200 regardless of the path. There's currently no feature to return a different response depending on the path.